supabase/migrations/20240603110253_init.sql

```diff
@@ -0,0 +1,63 @@
+create table "public"."repositories" (
+    "id" bigint generated by default as identity not null,
+    "created_at" timestamp with time zone not null default now(),
+    "url" character varying not null,
+    "last_reminder" timestamp with time zone default null,
+    "last_check" timestamp with time zone not null default now(),
+    "deadline" timestamp with time zone not null
+);
+
+
+alter table "public"."repositories" enable row level security;
+
+CREATE UNIQUE INDEX repositories_pkey ON public.repositories USING btree (id);
+
+CREATE UNIQUE INDEX repositories_url_key ON public.repositories USING btree (url);
+
+alter table "public"."repositories" add constraint "repositories_pkey" PRIMARY KEY using index "repositories_pkey";
+
+alter table "public"."repositories" add constraint "repositories_url_key" UNIQUE using index "repositories_url_key";
+
+grant delete on table "public"."repositories" to "anon";
```

Comment:
@gentlementlegen Why do we need to grant so many sensitive (delete, insert, update, etc...) permissions to the `anon` users?

Reply:
I removed them since they were clearly not needed.